src/main/kotlin/com/exactpro/th2/httpserver/server/options/Th2ServerOptions.kt

```diff
@@ -97,8 +98,8 @@ class Th2ServerOptions(
         }
     }
 
-    override fun onRequest(request: RawHttpRequest, uuid: String, parentEventID: String) {
-        val rawMessage = request.toRawMessage(connectionID, generateSequenceRequest(), uuid, parentEventID)
+    override fun onRequest(request: RawHttpRequest, uuid: String, parentEventID: EventID) {
+        val rawMessage = request.toRawMessage(connectionID, generateSequenceRequest(), uuid, parentEventID.id)
```

Comment:
```suggestion
        val rawMessage = request.toRawMessage(connectionID, generateSequenceRequest(), uuid, parentEventID)
```
We must pass full event id because it is composite